src/android/FileTransferBackground.java

```diff
@@ -126,23 +138,52 @@ public boolean execute(String action, JSONArray args, final CallbackContext call
         return true;
     }
 
+    private void createNotificationChannel() {
+        if (Build.VERSION.SDK_INT >= 26) {
+            NotificationChannel channel =new  NotificationChannel(
+                    "com.spoon.backgroundfileupload.channel",
+                    "upload channel",
+                    NotificationManager.IMPORTANCE_LOW
+            );
+            NotificationManager manager = (NotificationManager) cordova.getActivity()
+                    .getApplication()
```

Comment:
```suggestion
                .getApplication()
```
idem pour autres indentations